src/qibolab/instruments/zhinst.py

```diff
@@ -1359,7 +1413,7 @@ def split_batches(self, sequences):
     def play_sim(self, qubits, sequence, options, sim_time):
         """Play pulse sequence."""
 
-        self.experiment_flow(qubits, sequence, options)
+        self.experiment_flow(qubits, sequence, options)  # missing couplers?
```

Comment:
haha, does this call even work? :D I suppose no, since all necessary positional arguments are not passed to the function call.

Shall we remove this `play_sim` method since it is apparently not used anywhere (otherwise we would know that it is broken), or shall we fix it? @Jacfomg perhaps you have an opinion?

Reply:
Yeah, the `play_sim` stopped working some time ago and I didn't find the time or reasons to fix it. We could remove it and if we see the need of it we could reimplement it properly as if I remenber correctly it was basically passing the laboneq experiment object to a simulation function.